### PR TITLE
Add arbitrage trader labels for ethereal

### PR DESCRIPTION
**Pull request by AlexTheLion123**
Brief comments on the purpose of your changes:

Add labels for all wallets engaging in arbitrage activity on ethereal

*For Dune Engine V2*
I've checked that:
General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if adding a new model, I edited the alter table macro to display new database object (table or view) in UI explorer
* [ ] if adding a new materialized table, I edited the optimize table macro

Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: models/labels/arb_traders/labels_arb_traders_ethereum.sql [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      AND t1.evt_index != t2.evt_index
+    GROUP BY
+      t1.tx_from
+    ORDER BY
+      1
+  )
+select
+  "ethereum" as blockchain,
+  address,
+  "arb trader" AS name,
```
Comment (soispoke) [inline models/labels/arb_traders/labels_arb_traders_ethereum.sql:26]: I would suggest changing `arb trader` to `Arbitrage Trader` for label's name

File: models/labels/arb_traders/labels_arb_traders_schema.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    config:
+      tags: ['labels', 'ethereum', 'arbitrage', 'dex']
+    description: "Known arbitrage trader addresses across chains"
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "Blockchain"   
+      - &address
+        name: address
+        description: "Address of known arbitrage trader"
```
Comment (soispoke) [inline models/labels/arb_traders/labels_arb_traders_schema.yml:19]: Please make sure you add a name and description for all columns

File: models/labels/arb_traders/labels_arb_traders_ethereum.sql [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{{config(alias='arb_traders_ethereum')}}
```
Comment (soispoke) [inline models/labels/arb_traders/labels_arb_traders_ethereum.sql:1]: This model needs to be moved under the models/labels/arb_trades/ethereum directory

File: macros/alter_table_properties.sql [hunk 4]
```diff
@@ -307,6 +307,15 @@ ALTER TABLE labels.all SET TBLPROPERTIES('dune.public'='true',
                                         'dune.data_explorer.contributors'='["soispoke","hildobby"]');
 {% endset %}
 
+{% set labels_arb_traders %}
+ALTER VIEW labels.nft SET TBLPROPERTIES('dune.public'='true',
```
Comment (soispoke) [inline macros/alter_table_properties.sql:311]: `ALTER VIEW labels.nft` needs to be changed into `ALTER TABLE labels.arb_traders` here

**Review by soispoke — CHANGES_REQUESTED** [review]
Thanks @AlexTheLion123 for your submission

**Comment by jeff-dude** [comment]
hi @AlexTheLion123 i've tagged this PR as ready for review. i'll revisit asap. let me know here if anything changes.

**Comment by AlexTheLion123** [comment]
hi @jeff-dude. I've been working on a much more accurate query for arbitrage traders. Will update soon. Current PR is not accurate, apologies.

**Comment by jeff-dude** [comment]
no problem! good to know. i'm going to close this one, so we can start fresh :)

**Pull request closed without merging**